automation/taskcluster/lib/tasks.py

```diff
@@ -46,17 +47,19 @@ def craft_assemble_release_task(self, architectures, build_type, is_staging, ver
         artifacts = {
             'public/build/{}/target.apk'.format(arch): {
                 "type": 'file',
-                "path": '/opt/fenix/app/build/outputs/apk/'
-                        '{arch}/{build_type}/app-{arch}-{build_type}-unsigned.apk'.format(arch=arch, build_type=build_type),
+                "path": '/opt/{projec}/app/build/outputs/apk/'
```

Comment:
Typo here: `{project}`.
Is the purpose of having a single variable to make copy-pasting between projects easier?
Just within the Fenix project, this is adding some indirection (and extra weight in code reviews, since we need to make sure that new PRs are being consistent and using the `_PROJECT_NAME` variable instead of hard-coding)

Reply:
Fixed typo.

Yeah, copy-pasting and patch imports too. I don't mind reverting the change, if needed .

Reply:
I definitely understand the purpose of making it more copy-pastable, considering how identical it is to `reference-browser`.

However, since we don't/can't validate the copy-pastability, I'm mildly worried about us copy-pasting code that _appears_ ready to be directly moved, but some small chunk of it has `"fenix"` hardcoded in it that snuck past code review.
When we hard-code the project name everywhere, then we _know_ when we copy-paste to find-and-replace it.

:thinking: I'm not going to block the PR on this because I can see the value, I'm just worried about copy-pasting working 95% of the time, but at some point we'll confidently land code without testing it, and it'll fail in production.

Reply:
Got it. I reverted this commit.